src/modules/mavlink/mavlink_main.cpp

```diff
@@ -2220,13 +2115,10 @@ Mavlink::task_main(int argc, char *argv[])
 		 * make space for two messages plus off-by-one space as we use the empty element
 		 * marker ring buffer approach.
 		 */
-		if (OK != message_buffer_init(2 * sizeof(mavlink_message_t) + 1)) {
+		if (!_message_buffer.allocate(2 * sizeof(mavlink_message_t) + 1)) {
```

Comment:
update comment above "as we use the empty element marker ring buffer approach"

Reply:
To what would you change it? It's still true, isn't it?

Reply:
Nevermind, I guess it is the same implementation.